airflow/providers/amazon/aws/hooks/ecs.py

```diff
@@ -55,7 +55,24 @@ def should_retry_eni(exception: Exception):
     return False
 
 
-class EcsClusterStates(str, Enum):
+class _StringCompareEnum(Enum):
```

Comment:
Should we put this in a more general place so that other modules can make use of it?

Reply:
Right now it just for fix but in general we could move it later if it useful for other places, that why I make it "private"

Reply:
I was thinking the same thing.  Seems like something we'll want to reuse.

Reply:
Agree. We might want to do it now? It'd encourage others to use if needed instead of copy pasting it

Reply:
I feel like it is quite trivial to move it to a more generic place (and still keep it private if you'd prefer that). Seems easy to make the right choice now rather than having to fix it later. But I wont press any harder, happy to merge this as is and I can fix it in a follow up PR :+1:

Reply:
I'd rather move it in separate PR when we actually need this functional. Just for avoid spending time to choose better name and better place 🤣